tuxemon/core/components/config.py

```diff
@@ -83,7 +83,7 @@ def __init__(self, config_path=None):
         self.music_volume = cfg.getfloat("sound", "music_volume")
 
         # [game]
-        self.data = cfg.get("game", "data")
+        self.game = cfg.get("game", "game")
```

Comment:
please do not change this.  the word "game" has a specific meaning in the code and this will be confusing.

Reply:
Will revert in a following commit.